src/Umbraco.Forms.Integrations.Crm.Hubspot/HubspotWorkflow.cs

```diff
@@ -36,10 +36,10 @@ public HubspotWorkflow(ILogger<HubspotWorkflow> logger, IContactService contactS
             Group = "CRM";
         }
 
-        [Setting("Field Mappings", Description = "Map Umbraco Form fields to HubSpot contact fields", View = "~/App_Plugins/UmbracoForms.Integrations/Crm/Hubspot/hubspotfields.html")]
+        [Setting("Field Mappings", Description = "Map Umbraco Form fields to HubSpot contact fields", View = "Hubspot.PropertyEditorUi.Mapping")]
         public string FieldMappings { get; set; }
 
-        public override WorkflowExecutionStatus Execute(WorkflowExecutionContext context)
+        public WorkflowExecutionStatus Execute(WorkflowExecutionContext context)
```

Comment:
I think we should switch to the `async` Execute method and make service calls async.
So this 
```
var commandResult = _contactService.PostContactAsync(context.Record, fieldMappings, null).GetAwaiter().GetResult();
```
will be 
```
var commandResult = await _contactService.PostContactAsync(context.Record, fieldMappings, null);
```